ansible/provision-private-proxied-cluster-airgap-deb.yml

```diff
@@ -0,0 +1,65 @@
+### This is not a truly airgapped cluster!!! The intent here is specifically to test the install flow for an airgapped install
```

Comment:
Can we change this to something like:

> ### This is a semi-airgapped cluster deployment. The cluster can only access the internet via a proxied connection. This is currently only used for enabling download of Redpanda packages. If you have a centralized proxy, you may substitute that for the configured squid instance.

probably need to identify how to tell how to disable the squid install also

like need to do this for both deb/rpm files